.spelling

```diff
@@ -442,6 +442,7 @@ git
 GitHub
 GitOps
 GKE
+global_downstream_max_connections
```

Comment:
This should not be in .spelling. Use backticks around it in the doc.

Reply:
addressed in fb20f24f33984227d60f4a5e6a83cef44b50a808